scripts/longtermeffects.lua

```diff
@@ -11,19 +11,19 @@ function advanceRoundsOnTimeChanged(nRounds)
 			for _,nodeEffect in pairs(DB.getChildren(nodeCT, 'effects')) do
 				local nActive = DB.getValue(nodeEffect, 'isactive', 0);
 				if nActive ~= 0 then
-					local nodeCT = nodeEffect.getChild('...');
+					local nodeActor = nodeEffect.getChild('...');
 					local nDuration = DB.getValue(nodeEffect, 'duration');
 					local bHasDuration = (nDuration and nDuration ~= 0);
 					if bHasDuration and (nDuration <= nRounds) then
-						EffectManager.expireEffect(nodeCT, nodeEffect);
+						EffectManager.expireEffect(nodeActor, nodeEffect);
```

Comment:
Linter fix for variable shadowing of nodeCT from outer loop.

Reply:
technically not a problem, but definitely better to use a different variable name as you do here.
thanks!